client/src/lib.rs

```diff
@@ -494,6 +494,17 @@ impl Client {
     pub fn options(&self) -> &ClientOptions {
         &self.inner.options
     }
+
+    /// Return the options this client was initialized with mutably
+    pub fn options_mut(&mut self) -> &mut ClientOptions {
+        &mut self.inner.options
+    }
+
+    /// Set a worker build id to be attached to relevant requests. Unlikely to be useful outside
+    /// of core.
+    pub fn set_worker_build_id(&mut self, id: String) {
+        self.bound_worker_build_id = Some(id)
+    }
```

Comment:
Do you not plan on sharing this Client between different workers?

Reply:
All the important stuff is shared. Cloning them cones the underlying channel.